christfellowshipchurch/web-app-v2
# Added Person Guid to Analytics Tracking

### About
This PR adds the person `guid` prop when querying a the `currentUser`. This will help us with analytics our analytics on our website and tracking users back to their Rock profile.

**Other PRs:** https://github.com/christfellowshipchurch/services/pull/58

### Test Instructions
* run this [branch](https://github.com/christfellowshipchurch/services/pull/58) on the API with the website locally
* Add a `console.log({currentUser})` inside of the `useEffect()` method inside of the `AnalyticsProvider` to see the new `guid` property coming in.

### Screenshots
![image](https://github.com/christfellowshipchurch/web-app-v2/assets/46049974/0570ea60-0546-415c-a2ec-90af3174770f)

### Closes Tickets
[CFDP-2641](https://christfellowshipchurch.atlassian.net/browse/CFDP-2641)


[CFDP-2641]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2641?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
